libbeat/docs/dashboards.asciidoc

```diff
@@ -28,25 +28,32 @@ Make sure Kibana is running before you perform this step. If you are accessing a
 secured Kibana instance, make sure you've configured credentials as described in
 <<{beatname_lc}-configuration>>.
 
-To set up the Kibana dashboards for {beatname_uc}:
+To set up the Kibana dashboards for {beatname_uc}, use the appropriate command
+for your system.
 
 ifdef::allplatforms[]
 
-*deb, rpm, and mac:*
+ifeval::["{requires-sudo}"=="yes"]
 
-From the directory where you installed {beatname_uc}, run:
+include::../../libbeat/docs/shared-note-sudo.asciidoc[]
 
-["source","sh",subs="attributes,callouts"]
+endif::[]
+
+*deb and rpm:*
+
+["source","sh",subs="attributes"]
 ----------------------------------------------------------------------
-./{beatname_lc} setup --dashboards
+{beatname_lc} setup --dashboards
 ----------------------------------------------------------------------
 
-ifeval::["{requires-sudo}"=="yes"]
 
-If you changed ownership of the config file to root, you'll need preface this
-command with `sudo`.
+*mac:*
```

Comment:
This is also valid for Linux on `tar.gz` scenarios, maybe we should say `tar.gz` instead of `mac`?

Reply:
I'll just say mac and linux to match what we say on the product download pages.

Reply:
@exekias Actually, if I add linux to the topics here, I should also add Linux to the installation steps in the getting started. Then I'll have to decide whether I want to cover 32 or 64 bit. The docs start to get more complicated when we cover more platforms. I'm not saying we shouldn't...just that there are implications. WDYT?

Reply:
I think if we specify `mac` we will need linux as this is still the most common platform I think. I wonder if we could build this in a more modular way as it will be pretty tricky to keep it all up-to-date.